src/common/paths/write.yml

```diff
@@ -114,7 +114,13 @@ post:
     "404":
       $ref: "../../common/responses/ResourceNotFoundError.yml"
     "413":
-      description: Request entity too large. The payload exceeded the 50MB size limit. InfluxDB rejected the batch and did not write any data.
+      description: >
```

Comment:
I think we could do better here with more feature-ful yaml generation, but for now it's quite valuable for us to enforce that API's that are shared by cloud and OSS have the api specification in the same place.

This affects how we do codegen for common tooling (i.e. the CLI). If we allow the configurations to split it is quite easy for them to drift.